Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -59,7 +59,10 @@ struct TextureMapperLayer::ComputeTransformData {
     }
 };
 
-TextureMapperLayer::TextureMapperLayer() = default;
+TextureMapperLayer::TextureMapperLayer(const Settings& settings)
+    : m_settings(settings)
+{
+}
```

Comment:
This is a layering violation, platform should not have access to Settings from WebCore, we need a way to pass the two settings needed as bool or flags or whatever we want.

Reply:
Fixed, the `WebCore::Settings` reference is no longer passed down to `TextureMapperLayer`.